### DISCUSSION TITLE
Toggle for main display

### DISCUSSION THREAD

**Discussion by ginomeee** [xanderfrangos/twinkle-tray#420] [Ideas]
Add a **checkbox** for **toggling** which **display to be chosen as the main display**.

The use-case is mainly for people that often change workplaces and dock their laptops back to their external monitors in different places with different setups. Windows by default remembers the last setting but going back to Windows display settings and picking a display and toggling it as the main display is more cumbersome than just having a checkbox beside the brightness toggle which I now use daily. 

Sometimes I want my laptop display as my primary display and sometimes I want my external monitor or projector as my main display, it really does often change. Mainly this is because games only launch on the Windows primary display and clicking on the winkey only opens the start menu on the primary display.

Hope you guys could consider this, thanks!

**Comment by xanderfrangos** [comment]
Hi @ginomeee,

Thanks for the suggestion! This is something I'll consider for the future, but at this time it's outside of the scope of what I want Twinkle Tray to do.
